packages/formstr-app/src/containers/CreateForm/providers/FormBuilder/typeDefs.ts

```diff
@@ -2,8 +2,19 @@ import { AnswerTypes } from "@formstr/sdk/dist/interfaces";
 import { IQuestion } from "../../typeDefs";
 
 export interface IFormBuilderContext {
-  questionsList: IQuestion[],
-  saveForm: () => void,
-  editQuestion: (question: IQuestion, tempId: string) => void,
-  addQuestion: (answerType?: AnswerTypes) => void
-}
\ No newline at end of file
+  questionsList: IQuestion[];
+  saveForm: () => void;
+  editQuestion: (question: IQuestion, tempId: string) => void;
+  addQuestion: (answerType?: AnswerTypes) => void;
+  deleteQuestion: (tempId: string) => void;
+  questionIdInFocus?: string;
+  setQuestionIdInFocus: (tempId?: string) => void;
+  formSettings: IFormSettings;
+  updateFormSetting: (settings: IFormSettings) => void;
+}
+
+export interface IFormSettings {
```

Comment:
nice, should also make the corresponding changes in formstr sdk, maybe we can pick the interface from there directly